modules/authentication/proc-enabling-user-authentication-with-microsoft-azure-with-optional-steps.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="enabling-user-authentication-with-microsoft-azure"]
+= Enabling user authentication with {azure-brand-name}
```

Comment:
we need to make sure you can see the difference between the two azure procedures (mandatory steps vs + optional steps) straight away